### DISCUSSION TITLE
Non Technical Web3 roles

### DISCUSSION THREAD

**Discussion by ThembinkosiThemba** [Cyfrin/security-and-auditing-full-course-s23#185] [Q&A]
Hi, I need guidance. I am able to be technical and write smart contracts but I don't enjoy it that much, however, I enjoy doing more research, reading articles etc so I want wondering what more should I focus on?

**Comment by EngrPips** [comment]
Honestly, My best advice would be for you to incorporate researching into your technical career in any way that makes sense most as that would make you enjoy your technical role
